Documentation/instruction_name_tables.md

```diff
@@ -0,0 +1,62 @@
+## Database table naming rules
+
+Before uploading a new dataset in the database and creating a new table, the following rules should be followed to name the table:
+
+### All names:
+1. Always in lower case; 
+2. No space between the characters. Instead of using space, use `_` to separate composite names.
+3. Column names should follow the above rules.
+
+
+### Schema names:
+
+If the table it's single country data use the full ISO country name. This name for each country of the world can be found [here](https://gadm.org/maps.html).
```

Comment:
```suggestion
If the table is related to a single country data, use the full ISO country name. The name for each country can be found [here](https://gadm.org/maps.html).
```

Reply:
it would be good to give some examples here (also using `_` instead of whitespace, and using lowercase for the entire name).

Reply:
Agreed on giving some examples